src/utils/importFuncGenerator.ts

```diff
@@ -64,12 +64,12 @@ export function createImport(
     );
   const createStructImport = () => createCairoImportStructDefinition(name, path, ast, sourceUnit);
 
-  const warplibFunc = warplibImportInfo.get(path)?.get(name);
+  const warplibFunc = warplibImportInfo.get(path.join('.'))?.get(name);
   if (warplibFunc !== undefined) {
     return createFuncImport(...warplibFunc);
   }
 
-  switch (path + name) {
+  switch ([...path, name].join('.')) {
     case STARKWARE_CAIRO_COMMON_ALLOC + 'alloc':
```

Comment:
Let's import from `importFuncs` and compare it as arrays instead

Reply:
I hadn't given too much thought about this as Implicits should be remove soon, but it's true, this can be improved as well

Reply:
I'll add my constant phrase: lodash would fix it :D [link](https://lodash.com/docs/#isEqual)

Reply:
Do you mean having a Trie of paths? I think the switch case statement will live as long we update the code for not to need implicits. Which can be done at anytime, our Cairo1-ERC20 code already does not use implicits.

Reply:
I am of the opinion of leaving as it is, since it works, and it won't exist soon. Dev time can be better spent somewhere else

Reply:
If that's temporary and we won't need it in future then we can use strings, whatever is faster to implement for you @cicr99